### PR TITLE
Added PT (Portuguese) translations (#646)

### PR DESCRIPTION
**Pull request by ironmagician**
Translated everything that needed translations in portuguese.
This includes the messages listed in #646 and everything that was ready to be translated (containing 'lang: en')

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -3334,6 +3398,8 @@ plugins:
             text: 'After Character Creation you should deactivate and/or uninstall this.'
           - lang: de
             text: 'Nach der Charakter Erstellung sollten Sie es deaktivieren und/oder deinstallieren.'
+          - lang: pt
+          text: 'Após Criação de Personagem você deve desativar e/ou desinstalar isto.'
```
Comment (MacSplody) [inline masterlist.yaml]: Just needs to be indented by two spaces.

File: masterlist.yaml [hunk 2]
```diff
@@ -159,6 +184,9 @@ common:
         text: '%1%が最新版ではないようです。一部のModを動作させるには最新版の%1%で追加された機能が必要となる場合があります。'
       - lang: de
         text: 'Sie haben anscheinend nicht die aktuellste Version von %1% installiert. Einige Mods könnten Funktionalitäten benötigen, die nur in der aktuellsten Version von %1% enthalten sind, um zu funktionieren.'
+      - lang: pt
+        text: 'Aparenta que você não tem a versão mais recente de %1%. Alguns mods podem requerir funcinalidades adicionadas na última versão de %1% para funcionar.'
+
```
Comment (MacSplody) [inline masterlist.yaml]: The newline is fine, there's some extra white space, just remove the indent at the start of the line.


image

**Review by MacSplody — CHANGES_REQUESTED** [review]
Looks good only for two minor formatting issues.
Just correct those and it can be merged.

**Comment by ironmagician** [comment]
Thank you for the review. Fixed those up.

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
@ironmagician Thanks for the contribution.

**Pull request merged**